tests/test_calibration.py

```diff
@@ -225,15 +227,29 @@ def test_calibration_output():
     assert np.abs(image_dark_sub_old-image_dark_sub).all()<1e-3
     #assert np.abs(image_calib_nonflipped_old-image_calib_nonflipped).all()<1e-3
 
+def test_photometer():
+    
+    photometer_calib = Photometer("tests/calibration_data_test.toml")
+    with open('testdata/photometer_test_data_in.pkl', 'rb') as f:
+        photometer_data = pickle.load(f)
+    
+    photometer.calibrate_pm(photometer_data,photometer_calib)
+
+    with open('testdata/photometer_test_data_out.pkl', 'rb') as f:
+        photometer_data_out = pickle.load(f)
+    
+    for j in range(10):
+        for i in range(len(photometer_data.iloc[j])): assert(np.any(photometer_data.iloc[j][i]== photometer_data_out.iloc[j][i]))
 
 if __name__ == "__main__":
 
-    test_calibrate()
-    test_calibration_output() 
-    test_readfunctions()
-    test_CCDunit()
-    test_non_linearity_fullframe()
-    test_non_linearity_binned()
-    test_calibrate()
-    test_error_algebra()
-    test_channel_quaterion()
\ No newline at end of file
+    # test_calibrate()
+    # test_calibration_output() 
+    # test_readfunctions()
+    # test_CCDunit()
+    # test_non_linearity_fullframe()
+    # test_non_linearity_binned()
+    # test_calibrate()
+    # test_error_algebra()
+    # test_channel_quaterion()
+    test_photometer()
```

Comment:
Isn't it better just removing this `if` and using a test runner such as tox?

Reply:
Never used tox. The reason I have this is so that I can simply execute the test file in debug mode in order to debug the testing.

Reply:
nah Ill leave them ;)